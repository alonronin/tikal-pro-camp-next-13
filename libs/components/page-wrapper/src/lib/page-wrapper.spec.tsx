import { render } from '@testing-library/react';

import PageWrapper from './page-wrapper';

describe('ComponentsPageWrapper', () => {
  it('should render successfully', () => {
    const { container } = render(
      <PageWrapper>
        <h1>Hello World</h1>
      </PageWrapper>
    );
    expect(container).toMatchSnapshot();
  });
});
