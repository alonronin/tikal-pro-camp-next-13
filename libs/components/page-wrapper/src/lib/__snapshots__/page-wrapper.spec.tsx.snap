// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ComponentsPageWrapper should render successfully 1`] = `
<div>
  <div
    class="flex flex-col gap-4"
  >
    <h1>
      Hello World
    </h1>
  </div>
</div>
`;
